app/code/Magento/Tax/Block/Sales/Order/Tax.php

```diff
@@ -97,13 +97,13 @@ public function initTotals()
     /**
      * Add tax total string
      *
-     * @param string $after
+     * @param string $before
      * @return \Magento\Tax\Block\Sales\Order\Tax
      */
-    protected function _addTax($after = 'discount')
+    protected function _addTax($before = 'grand_total')
```

Comment:
https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/#modifying-the-default-values-of-optional-arguments-in-public-and-protected-methods

Can we just fix a place where this method is called?

Reply:
ohh, now i see, how i can be so stupid,!) just change position string and this done.